src/api/decisions/validations/validations.service.ts

```diff
@@ -269,6 +277,15 @@ export class ValidationService {
     }
   }
 
+  private normalizeText(text: string): string {
+    return text
+      ?.trim()
+      ?.replace(/[\[\]()]/g, '')
+      ?.replace(/[''"]/g, '')
+      ?.replace(/\s+/g, ' ')
+      ?.trim();
```

Comment:
I suck at regex, but I think you can shorten it to:
```
text
  ?.trim()
  ?.replace(/[\[\]()'"‘’“”]/g, '')
  ?.replace(/\s+/g, ' ');
```
Not sure you need any of the double `trim` functions on any of the `cleanValue` functions either.